.github/workflows/build_and_test.yml

```
@@ -82,7 +82,7 @@ jobs:
       run: |
         cd OpenDDS/ACE_TAO
         export ACE_COMMIT=$(git rev-parse HEAD)
-        echo "ACE_COMMIT=$ACE_COMMIT" >> $GITHUB_ENV
+        echo "ACE_COMMIT=$ACE_COMMIT" >> $GITHUB_ENV #??
```

```suggestion
        echo "ACE_COMMIT=$ACE_COMMIT" >> $GITHUB_ENV
```

https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable